doc/cephfs/standby.rst

```diff
@@ -83,10 +83,10 @@ will behave while in standby:
 
 ::
 
+    mds_standby_replay
     mds_standby_for_name
     mds_standby_for_rank
     mds_standby_for_fscid
-    mds_standby_replay
```

Comment:
Why?

Reply:
That is the sequence followed by the descriptions below.  Fixed just for easy reading.